kpops/utils/yaml.py

```diff
@@ -24,6 +27,9 @@ def load_yaml_file(
     file_path: Path, *, substitution: Mapping[str, Any] | None = None
 ) -> dict | list[dict]:
     with file_path.open() as yaml_file:
+        log.debug(
+            f"Picked up: {file_path.resolve().relative_to(file_path.resolve().parents[1])}"
```

Comment:
I have a question about the relative path. What happens if the pipeline folder is nested deeper? e.g. `pipelines/foo/bar/.../baz/pipeline.yaml`
would it only print `baz/pipeline.yaml`? imo relative to cwd as we normally do makes the most sense